pysnc/record.py

```diff
@@ -764,7 +808,22 @@ def set_display_value(self, field, value):
         else:
             c[field].set_display_value(value)
 
-    def get_link(self, no_stack=False) -> str:
+    def set_link(self, field, value):
+        """
+        Set the link for a field.
+
+        :param str field: The field
+        :param value: The Value
+        """
+        c = self._current()
+        if c is None:
+            raise NoRecordException('cannot get a value from nothing, did you forget to call next() or initialize()?')
+        if field not in c:
+            c[field] = GlideElement(field, link=value, parent_record=self)
+        else:
+            c[field].set_link(value)
+
+    def get_link(self, no_stack=False, field=None) -> str:
```

Comment:
I'm thinking if you want a link specific to a field, you should do it *from* the field -- aka don't bother having `field` parameter at the record level?

Reply:
Agree, this change felt like it has adding complexity/confusion, removed the `field` argument in favor of going after the element with `get_element` if you want to use the link.